### ISSUE TITLE
[ SUPPORT REQUEST ] - Game not launching

### ISSUE THREAD

**Issue by NinaMercer** [Just-Natsuki-Team/NatsukiModDev#620]
# Overview:
Exactly as the title says... the game doesn't launch, upon installing this.

## Additional information:
Upon attempting to launch, this appears in log.txt "Bootstrap to the start of init.init took 0.03s
Early init took 0.03s
Loader init took 0.09s
Loading error handling took 0.02s
Loading script took 0.86s
Loading persistent took 0.00s
Importing _renpysteam: ImportError('No module named _renpysteam',)" I've tried installing it a normal DDLC installation, right in common. I've tried taking the vanilla DDLC folder to the desktop, then installing JN. 

I have MAS, Monika After Story Installed.
I use the steam version of DDLC.
I use windows 10.
I just copy and pasted all the files in the JN folder, over to the DDLC folder. Including license and readme.md. If this has anything to do with it.

Do i have to get the version from DDLC's website, rather than the Steam version?

**Comment by NinaMercer** [comment]
Wait, nevermind. I didn't notice it said not to use the steam version, on the main page.

**Comment by RaionArt** [comment]
Does it run now, that you are using the steam version or do you still need assitance?

**Comment by NinaMercer** [comment]
> Does it run now, that you are using the steam version or do you still need assitance?

No, it doesn't run on the steam version. A file, or module is missing. Guess that's why it says not to use the steam version, but i downloaded the normal/official DDLC game. It works now, though i was expecting Monika to detect her...
